afisha-server/server/api/errors/errorBuilder.js

```diff
@@ -0,0 +1,10 @@
+const errorBuilder = (status, errorMessage, errorDescription) => {
+
+    return {
+        status: status,
+        message: errorMessage,
```

Comment:
What's different between error message and error description? Sounds like the same things, is it?